workstreams/supply-chain-maturity/README.md

```diff
@@ -0,0 +1,61 @@
+# Supply Chain Maturity Workstream README
```

Comment:
I think SCM as an acronym may be confusing, given our contexts. Proposed alternative: supply chain maturity model (SCMM) ?

Reply:
Naming will remain an unsolved problem for many years to come!
In the meantime, I've taken your suggestion.